Earthfile

```diff
@@ -27,23 +27,20 @@ package:
   COPY (./charts/*+package/*) /build/
   SAVE ARTIFACT /build AS LOCAL ./
 
-ci:  
-  BUILD +pre-commit
-  BUILD +tests # This target could depend on updated dependencies with the env variable NO_UPDATE
-  BUILD +package
-
 pre-commit:
   BUILD --pass-args +validate
   BUILD +template --TEMPLATE_FILE=readme.tpl --OUTPUT_FILE=README.md
   BUILD +template --TEMPLATE_FILE=contributing.tpl --OUTPUT_FILE=CONTRIBUTING.md
+  BUILD +tests # This target could depend on updated dependencies with the env variable NO_UPDATE
+  BUILD +package
 
 release:
   FROM core+builder-image
   GIT CLONE --branch=v1.6.1 https://github.com/helm/chart-releaser /src/chart-releaser
   WORKDIR /src/chart-releaser
   DO core+GO_INSTALL --package ./...
   COPY ./cr.yaml .
-  COPY (+ci/*) /build
+  COPY (+package/*) /build
```

Comment:
AS pre-commit must be done before we and +ci does not export any artifact anymore